tensilelite/Tensile/SolutionStructs.py

```diff
@@ -1359,12 +1361,12 @@ def assignProblemIndependentDerivedParameters(state):
 
       state["SubGroup0"]   = state["WorkGroup"][0]
       state["SubGroup1"]   = state["WorkGroup"][1]
-      state["LocalSplitU"] = state["WorkGroup"][2]
-
-    state["LocalSplitU"] = state["WorkGroup"][2]
+      # dot2: does not support LSU
```

Comment:
what if other MAC try to use LSU but not WaveSplitK

Reply:
I did not consider LSU yet in the current change. In that case, we may need a parameter to switch between these 2 modes. And this parameter should be put in kernel name.